python/docs/source/vowpalwabbit.dftovw.rst

```diff
@@ -0,0 +1,7 @@
+vowpalwabbit.dftovw
+====================
+
+.. automodule:: vowpalwabbit.dftovw
```

Comment:
i hope its not case sensitive

Reply:
I think it is?

Reply:
inconclusive , but 'it worked on my machine'